package proto_ff;

// 这里一定要包含自定义的协议
import "yd_fieldoptions.proto";
import "proto_common.proto";

message NameDesc
{
  optional uint64 id = 1 [(yd_fieldoptions.field_cname) = "编号",(yd_fieldoptions.db_field_comment) = "编号", (yd_fieldoptions.db_field_type) = E_FIELDTYPE_PRIMARYKEY, (yd_fieldoptions.db_field_auto_increment)=true, (yd_fieldoptions.db_field_auto_increment_value)=1];
  optional string name = 2[(yd_fieldoptions.field_cname) = "名字",(yd_fieldoptions.db_field_comment) = "名字", (yd_fieldoptions.db_field_type) = E_FIELDTYPE_INDEX, (yd_fieldoptions.field_bufsize)=64, (yd_fieldoptions.db_field_bufsize)=64];
}

message Sheet_NameDesc
{
  repeated NameDesc NameDesc_List = 1 [(yd_fieldoptions.field_arysize)=100];
}